install-steps.md

```diff
@@ -554,7 +553,47 @@ sudo cp ./oc /usr/local/bin/oc
 ~~~
 
+### Create RHCOS images cache (Optional)
+
+The installer needs to download two images, the RHCOS image used by the bootstrap VM and the RHCOS image used by the installer to provision the different nodes. Image caching is specially useful when running the installer on a network with limited bandwidth.
+
+When running the installer on a network with limited bandwidth if the RHCOS images download takes more than 15-20 minutes, the installer will timeout, having these images cached on a local `httpd` will help in such scenarios.
 
+The following steps will install `httpd` and download the required images:
+
+~~~sh
+# Install podman
+sudo dnf install -y podman
+# Open firewall port 8080 to be used for RHCOS Image caching
+sudo firewall-cmd --add-port=8080/tcp --zone=public --permanent
+# Create a directory to store the bootstraposimage and clusterosimage
+mkdir /home/kni/rhcos_image_cache
+# Set the appropriate SELinux context for the newly created directory
+sudo semanage fcontext -a -t httpd_sys_content_t "/home/kni/rhcos_image_cache(/.*)?"
+sudo restorecon -Rv rhcos_image_cache/
+# Get the commit id from the installer it will be used to determine which images do we need to download
+export COMMIT_ID=$(/home/kni/openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}')
+# Get the uri for the RHCOS image that will be deployed on the nodes
+export RHCOS_OPENSTACK_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.openstack.path | sed 's/"//g')
+# Get the uri for the RHCOS image that will be deployed on the bootstrap vm
+export RHCOS_QEMU_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.qemu.path | sed 's/"//g')
+# Get the path where the images are published
+export RHCOS_PATH=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json | jq .baseURI | sed 's/"//g')
+# Get the SHA hash for the RHCOS image that will be deployed on the bootstrap vm
+export RHCOS_QEMU_SHA_UNCOMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.qemu["uncompressed-sha256"]')
+# Get the SHA hash for the RHCOS image that will be deployed on the nodes
+export RHCOS_OPENSTACK_SHA_COMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.openstack.sha256')
+# Download the images and place them in the  /home/kni/rhcos_image_cache directory
+sudo curl -L ${RHCOS_PATH}${RHCOS_QEMU_URI} -o /home/kni/rhcos_image_cache
+sudo curl -L ${RHCOS_PATH}${RHCOS_OPENSTACK_URI} -o /home/kni/rhcos_image_cache
+# Confirm SELinux type is of httpd_sys_content_t for the newly created files.
+ls -Z /home/kni/rhcos_image_cache
+# Create pod
+sudo podman run -d --name rhcos_image_cache \
```

Comment:
lets remove sudo

Reply:
Aside from these 2 small changes above all looks good. can you squash your commits? and its ready to merge. thanks again for the PR!